ui/src/album/AlbumGridView.js

```diff
@@ -74,43 +108,32 @@ const Cover = withContentRect('bounds')(
 const AlbumGridTile = ({ showArtist, record, basePath }) => {
   const isDesktop = useMediaQuery((theme) => theme.breakpoints.up('md'))
   const classes = useStyles()
-  const [visible, setVisible] = useState(false)
 
   return (
-    <div
-      onMouseMove={() => {
-        setVisible(true)
-      }}
-      onMouseLeave={() => {
-        setVisible(false)
-      }}
-    >
-      <Link to={linkToRecord(basePath, record.id, 'show')}>
+    <div>
+      <Link
+        className={classes.link}
+        to={linkToRecord(basePath, record.id, 'show')}
+      >
         <Cover album={record} />
-        {(!isDesktop || visible) && (
-          <GridListTileBar
-            className={classes.tileBar}
-            title={record.name}
-            subtitle={
-              <div className={classes.albumArtistName}>
-                {showArtist ? (
```

Comment:
We should put this logic back, it makes the grid show Years instead of Artist name, when the grid is filtered by Artist (when it is showing an artist's discography)